packages/taraxa-ui/src/components/Card/index.ts

```diff
@@ -1 +1,3 @@
-export { default } from './Card';
+import Card from './Card';
+
+export default Card;
```

Comment:
Card component has no implementation yet?

Reply:
Card is already done as far as I looked.